core-feature-pack/src/main/resources/configuration/standalone/template.xml

```diff
@@ -46,7 +46,7 @@
             </logger>
         </audit-log>
         <management-interfaces>
-            <http-interface security-realm="ManagementRealm">
+            <http-interface security-realm="ManagementRealm" console-enabled="false">
```

Comment:
This is probably a good thing to do in general, but I'm a bit concerned changing this could affect someone in unexpected ways. (It doesn't affect full WildFly which has its own templates.) If we did this we'd want to change the host.xml and host-master.xml templates too.

The last commit at https://github.com/wildfly/wildfly-core/compare/master...bstansberry:pullRequest2934 results in admin-only behaving the same as normal when no console module is present.

Reply:
I agree, I think it is better if the fix is in the code rather than changing a template - an end user can always tweak the config effectively breaking the fix, if the fix is in the code it is always present.